tapchannel/aux_invoice_manager.go

```diff
@@ -175,13 +175,7 @@ func (s *AuxInvoiceManager) handleInvoiceAccept(_ context.Context,
 		return resp, nil
 	}
 
-	htlcBlob, err := req.WireCustomRecords.Serialize()
-	if err != nil {
-		return nil, fmt.Errorf("error serializing custom records: %w",
-			err)
-	}
-
-	htlc, err := rfqmsg.DecodeHtlc(htlcBlob)
+	htlc, err := rfqmsg.HtlcFromCustomRecords(req.WireCustomRecords)
```

Comment:
This is just a code improvement, right? `HtlcFromCustomRecords` wasn't available before, or we forgot to use it here.

Reply:
yeah same result, less lines, it's just all wrapped in the func above